src/cli.ts

```
@@ -447,113 +149,11 @@ export const arbRound = async (
-    }
-    if (options.quoteGas) {
-        try {
-            options.quoteGas = BigInt(options.quoteGas);
-        } catch {
-            throw "invalid quoteGas value, must be an integer greater than equal zero";
-        }
-    } else {
-        options.quoteGas = 1_000_000n; // default
-    }
-    const poolUpdateInterval = _poolUpdateInterval * 60 * 1000;
+    const poolUpdateInterval = options.poolUpdateInterval! * 60 * 1000;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)